Ginger/Ginger/AutomatePageLib/AddActionMenu/ApplicationModels/POMNavPage.xaml.cs

```diff
@@ -157,6 +160,11 @@ private void MainTreeView_ItemSelected(object sender, SelectionTreeEventArgs e)
                     xPomElementsListView.DataSourceList = mPOM.MappedUIElements;
                     xPomElementsListView.Visibility = Visibility.Visible;
                     xPOMSplitter.IsEnabled = true;
+                    //filre POM Activities to show
+                    ObservableList<Activity> sharedActivities = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<Activity>();
+                    ObservableList<Activity> pomActivities = GingerCore.General.ConvertListToObservableList(sharedActivities.Where(x => x.Guid == mPOM.ActivitiesGuid.FirstOrDefault(y => y == x.Guid)).ToList());
```

Comment:
change sharedActivities.Where(x => x.Guid == POM.ActivitiesGuid.FirstOrDefault(y => y == x.Guid)).ToList())
to
sharedActivities.Where(x => POM.ActivitiesGuid.Contains(x.Guid))